docker/Dockerfile.integration_test

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+LABEL mantainer="Vincenzo Palazzo vincenzopalazzodev@gmail.com"
+
+# Ubuntu utils
+RUN apt-get update && apt-get install -y \
+    software-properties-common wget && \
+    sh -c 'wget -qO- https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -' && \
+    apt-get update && apt install golang-go
+
+# Install bitcoin core and lightningd (last version)
+RUN add-apt-repository ppa:luke-jr/bitcoincore
+RUN apt-get update  && apt-get install -y bitcoind libsodium-dev libpq-dev jq make
+
+ENV CLIGHTNING_VERSION=0.10.2
+
+RUN wget https://github.com/ElementsProject/lightning/releases/download/v$CLIGHTNING_VERSION/clightning-v$CLIGHTNING_VERSION-Ubuntu-20.04.tar.xz && \
+    tar -xvf clightning-v$CLIGHTNING_VERSION-Ubuntu-20.04.tar.xz -C /opt && cd /opt && \
+    mv usr clightning-v$CLIGHTNING_VERSION
+
+RUN rm -r clightning-v$CLIGHTNING_VERSION-Ubuntu-20.04.tar.xz
+
+
+ENV PATH $PATH:/usr/local/go/bin
+ENV CLN_UNIX_SOCKET=/opt/clightning-v$CLIGHTNING_VERSION/bin:$PATH
+
+WORKDIR workdir
+
+CMD ["cd" "client"]
+
+CMD ["go" "test"]
```

Comment:
ok, this is good, but we can do better by adding a second script that will run each time that the container is run.

usually it is called `entrypoint.sh`